masonry/src/text/text_layout.rs

```diff
@@ -37,8 +37,7 @@ use crate::text::render_text;
 ///
 /// TODO: Update docs to mentionParley
 #[derive(Clone)]
-pub struct TextLayout<T> {
-    text: T,
+pub struct TextLayout {
     // TODO: Find a way to let this use borrowed data
```

Comment:
Is this comment still accurate?

Reply:
If you mean the "use borrowed data" comment, yes, it's still accurate.

It's from this commit: https://github.com/linebender/xilem/commit/732cfa8376d5e19fbdd98b4b73806f0c522e740e#diff-da6cde61a550125d612ef93c9aecd16c1210e2cfbfb664d3ba0a9ab00ddc2caf

Not sure what it means either. @DJMcNab ?

Reply:
I have no idea. It seems you understand this comment better than me - because I assume that it applied to the `text` field and was misplaced.

Reply:
We'll remove it then.